### DISCUSSION TITLE
Add behavior registration event on Controllers

### DISCUSSION THREAD

**Discussion by nateiler** [craftcms/cms#13476] [Ideas] [answered]
Similar to the base Model class, Controllers would also greatly benefit from allowing third parties to register behaviors against them.

A pull request to come.

**Comment by nateiler** [comment, marked as answer]
Pull request submitted: https://github.com/craftcms/cms/pull/13477

**Reply by brandonkelly** [reply]
Released in Craft 4.5.0!
